drivers/ethernet/eth_w5500.c

```diff
@@ -34,6 +34,7 @@ LOG_MODULE_REGISTER(eth_w5500, CONFIG_ETHERNET_LOG_LEVEL);
 #define W5500_SPI_WRITE_CONTROL(addr)   \
```

Comment:
Please change the commit subject to
`drivers: ethernet: w5500: Check int pin after processing interrupt`
Also there is line length limits in commit message body so you should split it to multiple lines (sanity checker will complain about those)